pkgs/development/python-modules/numpymaxflow/default.nix

```diff
@@ -0,0 +1,44 @@
+{
+  lib,
+  buildPythonPackage,
+  pythonOlder,
+  fetchFromGitHub,
+  python,
+  numpy,
+  setuptools,
+}:
+
+buildPythonPackage rec {
+  pname = "numpymaxflow";
+  version = "0.0.7";
+  pyproject = true;
+
+  disabled = pythonOlder "3.9";
+
+  src = fetchFromGitHub {
+    owner = "masadcv";
+    repo = "numpymaxflow";
+    rev = "refs/tags/v${version}";
+    hash = "sha256-s1Pbab3vK0Co1KQev+/4+IRqyLzqUXBixz4b0ndq+5k=";
+  };
+
+  postPatch = ''
+    substituteInPlace pyproject.toml --replace-fail "numpy==1.26.0" "numpy"
```

Comment:
If this is not a build-system dependency, then it could be relaxed via `pythonRelaxDeps`

Reply:
It is, a bit unfortunately.

Reply:
I don't see it in the `build-system` list below tho 🤔 It's listed as a normal dependency

Reply:
Missed it. Fixed.